cmd/compose/run.go

```diff
@@ -65,6 +67,7 @@ type runOptions struct {
 	ignoreOrphans bool
 	removeOrphans bool
 	quietPull     bool
+	EnvFiles      []string
```

Comment:
doest it need to be access somewhere else?
```suggestion
	envFiles      []string
```